javalib/src/main/scala/java/util/AbstractSequentialList.scala

```diff
@@ -0,0 +1,42 @@
+package java.util
+
+abstract class AbstractSequentialList[E] protected ()
+    extends AbstractList[E] {
+
+  def get(index: Int): E =
+    listIterator(index).next()
```

Comment:
I believe this would throw the wrong kind of exception if you call it as `list.get(list.size)`. As written, it will throw a `NoSuchElementException` because creating the iterator itself is valid for `list.size`, but then `next()` throws that. But the spec demands an `IndexOutOfBoundsException`.